src/playbacks/html5_audio/html5_audio.js

```diff
@@ -5,158 +5,32 @@
 import Playback from 'base/playback'
 import Events from 'base/events'
 import find from 'lodash.find'
+import HTML5Video from 'playbacks/html5_video'
 
-export default class HTML5Audio extends Playback {
+export default class HTML5Audio extends HTML5Video {
   get name() { return 'html5_audio' }
   get tagName() { return 'audio' }
-  get events() {
-    return {
-      'loadedmetadata': 'loadedMetadata',
-      'stalled': 'stalled',
-      'waiting': 'waiting',
-      'timeupdate': 'timeUpdated',
-      'ended': 'ended',
-      'canplaythrough': 'bufferFull',
-      'playing': 'playing',
-      'pause': 'paused'
-    }
-  }
-
-  constructor(params) {
-    super(params)
-    this.options = params
-    this.settings = {
-      left: ['playpause', 'position', 'duration'],
-      right: ['fullscreen', 'volume'],
-      default: ['seekbar'],
-      seekEnabled: true
-    }
-    this.render()
-    params.autoPlay && this.play()
-  }
-
-  bindEvents() {
-    this.listenTo(this.container, Events.CONTAINER_PLAY, this.play)
-    this.listenTo(this.container, Events.CONTAINER_PAUSE, this.pause)
-    this.listenTo(this.container, Events.CONTAINER_SEEK, this.seek)
-    this.listenTo(this.container, Events.CONTAINER_VOLUME, this.volume)
-    this.listenTo(this.container, Events.CONTAINER_STOP, this.stop)
-  }
-
-  loadedMetadata(e) {
-    this.durationChange()
-    this.trigger(Events.PLAYBACK_LOADEDMETADATA, e.target.duration)
-  }
 
   durationChange() {
-    // we can't figure out if hls resource is VoD or not until it is being loaded or duration has changed.
-    // that's why we check it again and update media control accordingly.
-    if (this.getPlaybackType() === 'aod') {
-      this.settings.left = ["playpause", "position", "duration"]
-    } else {
-      this.settings.left = ["playstop"]
-    }
-    this.settings.seekEnabled = isFinite(this.getDuration())
+    this.settings.left = ["playpause", "position", "duration"]
+    this.settings.seekEnabled = this.isSeekEnabled()
     this.trigger(Events.PLAYBACK_SETTINGSUPDATE)
   }
-
+  
   getPlaybackType() {
-    return [0, undefined, Infinity].indexOf(this.el.duration) >= 0 ? 'live' : 'aod'
+    return 'aod'
   }
 
   stalled() {
-    if (this.getPlaybackType() === 'vod' && this.el.readyState < this.el.HAVE_FUTURE_DATA) {
```

Comment:
Not sure why `this.getPlaybackType() === 'vod'` used to be there? Wouldn't it always have returned 'live' or 'aod'?? (line 64)